### PR TITLE
[Docs][NextJS] Embed videos

### PR DESCRIPTION
**Pull request by AncaIO**
Embeds video material based on recommendation from "Next.js YouTube Content" email: 
 
- https://www.youtube.com/watch?v=mBOPr4djFMg in Why JSS (the portion about Jamstack).
- https://www.youtube.com/watch?v=CfoOKlKjzfA in Why Next.js

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation update (non-breaking change; modified files are limited to the `/docs` directory)

## Checklist:

- [ ] I have read the Contributing guide.
- [ ] My code follows the code style of this project.
- [ ] My code/comments/docs fully adhere to the Code of Conduct.
- [ ] My change is a code change and it requires an update to the documentation.
- [ ] My change is a documentation change and it requires an update to the navigation.

### CODE REVIEW

File: docs/data/routes/docs/introduction/why-jss/en.md [hunk 1]
```diff
@@ -15,7 +15,13 @@ After components are assembled, Content Authors can apply Sitecore-specific feat
 ![Authoring example](/assets/img/authoring-example.png)
 
 ## What is JSS
-Initially, the only choice for implementing Sitecore was to use .NET. But in the past few years, JavaScript experienced a huge spike in popularity, and companies realized that [Jamstack](https://jamstack.org/) architecture is very beneficial. Sitecore recognizes the popularity of JavaScript frameworks and the benefits of building with Jamstack architecture. The primary goal of Sitecore JSS is to add first-class support for JavaScript as an implementation option for Sitecore-powered web apps.
+Initially, the only choice for implementing Sitecore was to use .NET. But in the past few years, JavaScript experienced a huge spike in popularity, and companies realized that [Jamstack](https://jamstack.org/) architecture is very beneficial. Sitecore recognizes the popularity of JavaScript frameworks and the benefits of building with Jamstack architecture. 
+
+<p>
+  <iframe width="672" height="378" src="https://www.youtube.com/embed/mBOPr4djFMg" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```
Comment (anastasiya29) [inline docs/data/routes/docs/introduction/why-jss/en.md:21]: This is a section about "What is JSS", and this video is about "What is Jamstack", which is not JSS-specific. Would it make more sense to move this to "Why JSS" or "Developer benefits of JSS"?

File: docs/data/routes/docs/nextjs/getting-started-nextjs/why-nextjs/en.md [hunk 2]
```diff
@@ -9,6 +9,10 @@ title: Why Next.js and JSS
 
 The latest JSS SDK leverages many of Next.js' [features](https://nextjs.org/#features), to simplify and enhance development workflow and experience. 
 
+<p>
+<iframe width="672" height="378" src="https://www.youtube.com/embed/CfoOKlKjzfA" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+</p>
+
```
Comment (anastasiya29) [inline docs/data/routes/docs/nextjs/getting-started-nextjs/why-nextjs/en.md:15]: Each video starts with a sentence that summarizes the video - would it be useful to write this sentence into the docs? Will people be more likely to start the video if they know what it's about?

Comment (anastasiya29) [inline docs/data/routes/docs/nextjs/getting-started-nextjs/why-nextjs/en.md:15]: Would it be useful to add a link to "more videos" somewhere?
https://www.youtube.com/playlist?list=PL1jJVFm_lGnwZup4L4BjITS2sKr4rpMfI

File: docs/data/routes/docs/introduction/why-jss/en.md (hunk 1 above)
Comment (AncaIO) [reply docs/data/routes/docs/introduction/why-jss/en.md:21]: @anastasiya29 Well, the Jamstack bit doesn't link at all with surrounding content, with or without the video. 

I propose we move the Jamstack bit in "Why JSS" and rework that section to something like: 

```
## Why JSS

Initially, the only choice for implementing Sitecore was to use .NET. But in the past few years, JavaScript experienced a massive spike in popularity, and companies realized the benefits of [Jamstack](https://jamstack.org/) architecture. 

<p>
  Watch the following video to understand what is and isn't Jamstack, and why it matters.
  <iframe width="672" height="378" src="https://www.youtube.com/embed/mBOPr4djFMg" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
</p>

Integrating a front-end framework with an authoring interface as flexible as Sitecore's is a significant undertaking and poses some challenges. Front-end frameworks require developers to build the UI component hierarchies in code, meaning developers control the component composition. A typical front-end workflow would, therefore, sacrifice some authoring features. However, in Sitecore, we want Content Authors to keep control, which means we cannot know the component hierarchy at build-time. (better emphasis of why JS framework are a challenge)

Sitecore JSS solves these challenges and adds first-class support for JavaScript frameworks (and Jamstack architecture) as an implementation option for Sitecore-powered web apps. (We plug in Jamstack)

With JSS, all the benefits of JavaScript frameworks, such as Next's image optimization and static site generation, can be utilized on top of Sitecore features.
```

What do you think?

**Review by anastasiya29 — APPROVED** [review]

**Pull request merged**